# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1305650479 25200
#      Tue May 17 09:41:19 2011 -0700
# Branch stable
# Node ID c21eb9c02c3139cbc7e201bbd0c505ea4edbb7d0
# Parent  c09ed93d1a8b8e39cb90a44b0427475982ab6047
frnd.m: Fix typo in 2-input calling form preventing execution.

* frnd.m: Replace typo 'a' with input variable 'm'.

diff --git a/scripts/statistics/distributions/frnd.m b/scripts/statistics/distributions/frnd.m
--- a/scripts/statistics/distributions/frnd.m
+++ b/scripts/statistics/distributions/frnd.m
@@ -66,17 +66,17 @@ function rnd = frnd (m, n, r, c)
       error ("frnd: R must be a positive integer or vector");
     endif
 
     if (any (size (m) != 1)
         && ((length (size (m)) != length (sz)) || any (size (m) != sz)))
       error ("frnd: M and N must be scalar or of size SZ");
     endif
   elseif (nargin == 2)
-    sz = size(a);
+    sz = size(m);
   else
     print_usage ();
   endif
 
 
   if (isscalar (m) && isscalar (n))
     if (isinf (m) || isinf (n))
       if (isinf (m))
